### PR TITLE
feat(node): v13.7

### PR DESCRIPTION
**Pull request by SimonSchick**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V13.md#13.7.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by SimonSchick** [comment]
Please note that I failed to update the version to `13.6` in my last PR, hence the jump in this one.

**Comment by SimonSchick** [comment]
Build failure is unrelated to changes.

File: types/node/module.d.ts [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+            originalColumn: number;
+        }
+
+        /**
+         * @experimental
+         */
+        class SourceMap {
+            readonly payload: SourceMapPayload;
+            constructor(payload: SourceMapPayload);
+            findEntry(line: number, column: number): SourceMapping;
```
Comment (Flarna) [inline types/node/module.d.ts]:
```suggestion
            findEntry(line: number, column: number): SourceMapping | {};
```
see https://github.com/nodejs/node/blob/bbab963db80528f3f932269774f1d5d6012fbb68/lib/internal/source_map/source_map.js#L195

File: types/node/module.d.ts [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
+        /**
+         * Updates all the live bindings for builtin ES Modules to match the properties of the CommonJS exports.
+         * It does not add or remove exported names from the ES Modules.
+         */
+        function syncBuiltinESMExports(): void;
+
+        /**
+         * @experimental
+         */
+        function findSourceMap(path: string, error?: Error): SourceMap;
```
Comment (ExE-Boss) [inline types/node/module.d.ts]: Shouldn’t&nbsp;these be&nbsp;defined as&nbsp;static&nbsp;members of&nbsp;the&nbsp;`Module`&nbsp;class?

File: types/node/module.d.ts [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+         * @experimental
+         */
+        class SourceMap {
+            readonly payload: SourceMapPayload;
+            constructor(payload: SourceMapPayload);
+            findEntry(line: number, column: number): SourceMapping;
+        }
+    }
     interface Module extends NodeModule {}
     class Module {
```
Comment (ExE-Boss) [inline types/node/module.d.ts]: Can’t&nbsp;this&nbsp;just be&nbsp;<code>class&nbsp;Module implements&nbsp;NodeModule</code>?

Comment (SimonSchick) [reply types/node/module.d.ts]: No, we would have to duplicate all properties in that case.
Also out of scope of this PR.

File: types/node/module.d.ts (hunk 2 above)
Comment (SimonSchick) [reply types/node/module.d.ts]: They should be both but similarly to `events` the exports are hard to model without duplicating a bunch of things.
It's also not _really_ intended to be used as a static but rather just slapped on the module for exports.
See https://github.com/nodejs/node/blob/master/lib/module.js#L7
For that reason I have decided to not model it as such.

**Comment by ExE-Boss** [comment]
`@types/assert` has&nbsp;now been&nbsp;fixed in&nbsp;<https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41916>.

**Review by ExE-Boss — APPROVED** [review]

**Review by galkin — APPROVED** [review]

File: types/node/module.d.ts (hunk 1 above)
Comment (SimonSchick) [reply types/node/module.d.ts]: @bcoe @MylesBorins for reference.

**Pull request merged**